kinds/accesspolicy/access_policy_kind.cue

```diff
@@ -0,0 +1,54 @@
+package kind
+
+name:              "AccessPolicy"
+maturity:          "merged"
+description:       "Access rules for a scope+role.  NOTE there is a unique constraint on role+scope"
+pluralName:        "AccessPolicies"
+machineName:       "accesspolicy"
+pluralMachineName: "accesspolicies"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				spec: {
+					// The scope where these policies should apply
+					scope: #ResourceRef
+
+					// The role that must apply this policy 
+					role: #RoleRef
+
+					// The set of rules to apply.  Note that * is required to modify
+					// access policy rules, and that "none" will reject all actions
+					rules: [...#AccessRule]
+				} @cuetsy(kind="interface")
+
+				#RoleRef: {
+					// Policies can apply to roles, teams, or users
+					// Applying policies to individual users is supported, but discouraged
+					kind:  "Role" | "BuiltinRole" | "Team" | "User"
+					name:  string
+					xname: string // temporary
```

Comment:
I added xname so we can easily see/audit the current identifier name rather than the uid